package.json

```diff
@@ -75,7 +75,7 @@
     "underscore.string": "^3.3.5",
     "user-home": "^2.0.0",
     "uuid": "^3.3.2",
-    "ws": "^7.0.1"
+    "ws": "6.1.4"
```

Comment:
is this not a breaking change?

Reply:
also can we make it non-fixed version?

Reply:
not breaking because we are using primus, which wraps ws, also this version is more permissive than 7, we could make it non-fixed, suppose we should just trust in version 6's patches.

Reply:
Okay just want some opinions on the team. @JensEggers @JohanvdWest @cyrussalem  Are we happy with this? I know we had issues with old ws deps being brought in by other modules in some projects and had to add a dep of 7 to make sure it is used by happn - if this gets updated without a breaking change it means we will end up with both 7 and 6 in those projects.

Reply:
@HaroldStruwig : apart from happn-stats, do you know of other modules that have this update? The other fix I looked at is have our happn-primus-wrapper create a transformer that intercepts incoming upgrade requests and adds required headers - I did some quick and dirty coding to confirm this was possible, but it will take some time to build and test properly.

Reply:
As discussed with the rest of the team it seems like this won't have any consequences in the way it is used